eng/pipelines/templates/stages/archetype-c-release.yml

```diff
@@ -0,0 +1,80 @@
+stages:
+- ${{if and(eq(variables['Build.Reason'], 'Manual'), eq(variables['System.TeamProject'], 'internal'))}}:
+  - stage: Release_C_SDK
+    displayName: 'Release C SDK'
+    dependsOn: Build
+
+    jobs:
+      - deployment: TagRepository
+        displayName: "Create release tag"
+        condition: ne(variables['Skip.TagRepository'], 'true')
+        environment: github
+
+        pool:
+          vmImage: windows-2019
+
+        strategy:
+          runOnce:
+            deploy:
+              steps:
+                - checkout: none
+                - pwsh: Get-ChildItem -Recurse $(Pipeline.Workspace)/release-info
+                  displayName: Output Visible Artifacts
+                - pwsh: >-
```

Comment:
Why are we not using the template? /eng/common/pipelines/templates/steps/create-tags-and-git-release.yml